docs/introduction.rst

```diff
@@ -1,10 +1,22 @@
 Introduction to Flatpak
 =======================
 
-Flatpak is a framework for distributing desktop applications on Linux. It
-has been created by developers who have a long history of working on the
+Flatpak is a framework for distributing desktop applications across various Linux
+distributions. It has been created by developers who have a long history of working on the
 Linux desktop, and is run as an independent open source project.
 
+Terminology
+-----------
+
+- Flatpak: The Flatpak package manager.
```

Comment:
Punctuation issue here: colons do not start new sentences, "The" therefore shouldn't be capitalised.

Reply:
Really? I was taught at school that they should be capitalized. I love English! /s